third-party/realdds/include/realdds/topics/image/image-msg.h

```diff
@@ -34,14 +43,32 @@ class image
 public:
     using type = raw::device::imagePubSubType;
 
+    image() = default;
     image( const raw::device::image & );
+    image( const image & );
```

Comment:
Why do we need to enable copying? This is dangerous in the case of images, as we're talking about a lot of data.
If possible, I recommend removing it.

Reply:
Removed